tensorflow_io/core/kernels/audio_kernels.h

```diff
@@ -45,5 +54,40 @@ Status MP4ReadableResourceInit(
     Env* env, const string& input,
     std::unique_ptr<AudioReadableResourceBase>& resource);
 
+// Container for decoded audio.
+class DecodedAudio {
+public:
```

Comment:
Not sure we need a `class` here, as it is just a `struct` with one function that does a 
```
channels * samples_perchannel * sizeof(int16);
```

maybe we don't need this class after all?

Reply:
Yes, I think you're right. I think a struct is enough.